apicast/Roverfile.lock

```diff
@@ -1,20 +1,23 @@
-busted 2.0.rc12-1
-dkjson 2.5-2
-inspect 3.1.0-1
-ldoc 1.4.6-2
-lua-resty-env 0.4.0-1
-lua-resty-http 0.10-0
-lua-resty-iputils 0.3.0-1
-lua-resty-jwt 0.1.10-1
-lua-resty-repl 0.0.6-0
-lua-resty-url 0.2.0-1
-lua-term 0.7-1
-lua_cliargs 3.0-1
-luafilesystem 1.7.0-2
-luassert 1.7.10-0
-luasystem 0.2.1-0
-markdown 0.33-1
-mediator_lua 1.1.2-0
-penlight 1.5.4-1
-router 2.1-0
-say 1.3-1
+argparse 0.5.0-1|
+busted 2.0.rc12-1|
+dkjson 2.5-2|
+inspect 3.1.0-1|
+ldoc 1.4.6-2|
+liquid scm-1|811a73e38fdd9fdea116be4baf310ca326b96c77
+lua-resty-env 0.4.0-1|
+lua-resty-execvp 0.1.0-1|
+lua-resty-http 0.11-0|
+lua-resty-iputils 0.3.0-1|
+lua-resty-jwt 0.1.11-0|
+lua-resty-repl 0.0.6-0|3878f41b7e8f97b1c96919db19dbee9496569dda
```

Comment:
Why the hash in this one? It has a version.

Reply:
Yes, but version can still point to a scm to get the source. Depends if it is a rock or a rockspec.

rock has a source packaged in the bundle but rockspec has to download it from somewhere
in this case repl has only rockspec and that points to a git tag https://luarocks.org/modules/saksmlz/lua-resty-repl/0.0.6-0